README.md

```diff
@@ -21,3 +21,63 @@ If your platform does not support one of the required features, it is also possi
 Testing requires access to an archive node for onchain quote comparisons. This can be done using Infura.
 
 `pnpm test`
+
+## Examples
+
+### Balancer Api Provider
+
+Joining with pool state:
```

Comment:
This example should be updated with the latest version of the code where JoinParser refactored.